src/picknik_ur_multi_arm_config/config/realsense_config_high_accuracy.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Again, does this file need to be here, or can we reuse the one from other configs?